### PR TITLE
Disable sampling test on Ruby

### PR DESCRIPTION
**Pull request by lloeki**

### CODE REVIEW

File: tests/test_sampling.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def next_request_id(cls):
     @bug(context.library < "java@0.92.0")
     @flaky(context.library < "python@0.57.0")
     @flaky(context.library >= "java@0.98.0", reason="APMJAVA-743")
+    @flaky(library="ruby", reason="fails randomly for Sinatra on JSON body that dutifully keeps")
```
Comment (cbeauchesne) [inline tests/test_sampling.py:34]:
```suggestion
    @flaky(library="ruby", weblog_variant="sinatra", reason="fails randomly for Sinatra on JSON body that dutifully keeps")
```

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/test_sampling.py (hunk 1 above)
Comment (cbeauchesne) [reply tests/test_sampling.py:34]: if it fails only on sinatra :)

Comment (lloeki) [reply tests/test_sampling.py:34]: Given it appears something is unfit in the test and there is no significant behaviour difference between Rails, Sinatra, and Rack except timing, I was keen on disabling the test altogether until it is not flaky anymore.

Comment (lloeki) [reply tests/test_sampling.py]: Applied anyway.

**Review by cbeauchesne — APPROVED** [review]

**Comment by lloeki** [comment]
Thanks!

**Pull request merged**